packages/vuetify/src/components/VCalendar/VCalendarDaily.ts

```diff
@@ -192,6 +192,10 @@ export default CalendarWithIntervals.extend({
           height,
           ...styler(interval),
         },
+        on: {
+          dragover: (event: DragEvent) => event.preventDefault(),
+          drop: (event: DragEvent) => this.$emit('drop:event', { event, interval }),
```

Comment:
- Wouldn't this be `drop:time`?
- Why not just add all the drag events to `getDefaultMouseEventHandlers`?